package.json

```diff
@@ -4,11 +4,26 @@
   "description": "",
   "main": "cypress.config.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "cy:open": "cypress open",
+    "cy:run": "cypress run",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix .",
+    "prepare": "husky install",
+    "prettier": "prettier --write \"**/*.{js,json,ts}\""
```

Comment:
"prettier": "prettier --write '**/**.{ts,js,tsx}' '*.{json,md,yml}'", what about yamls and readme! :D